src/scripts/main.js

```
@@ -355,6 +355,47 @@ const people = [
 ];
 
 // eslint-disable-next-line no-console
-console.log(people); // you can remove it
 
-// write your code here
+const table = document.querySelector('table');
+
+for (const person of people) {
+  class Person {
+    constructor() {
+      this.name = person.name;
+      this.gender = person.sex;
+      this.born = person.born;
+      this.died = person.died;
+      this.age = person.died - person.born;
+      this.century = Math.ceil(person.died / 100);
+
+      switch (this.gender) {
+        case 'm':
+          this.gender = 'Male';
+          break;
+        case 'f':
+          this.gender = 'Female';
+          break;
+        default:
+          return 0;
+      }
```

The `switch` statement could be simplified by directly assigning the gender based on the condition. For example:

```javascript
this.gender = this.gender === 'm' ? 'Male' : this.gender === 'f' ? 'Female' : null;
```

Also, consider throwing an error instead of `return 0` in the default case. Returning 0 doesn't provide much information about what went wrong.